### PR TITLE
checking in FAQ page

### PR DESCRIPTION
**Pull request by megalodonatron**
If you think I left anything out please let me know so I can add it :rabbit: fixes issue #280

### CODE REVIEW

**Comment by ElDeveloper** [comment]
webdev is down. Right now, I guess there is no good way to test this for now. :japanese_ogre: :finnadie:

**Comment by douginator2000** [comment]
Should be fine now - QB was down but is now back up.

On Mon, Apr 1, 2013 at 11:38 AM, Yoshiki Vázquez Baeza <
notifications@github.com> wrote:

> webdev is down. Right now, I guess there is no good way to test this for
> now. [image: :japanese_ogre:][image: :finnadie:]
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/qiime/qiime_web_app/pull/295#issuecomment-15726316
> .

## 

Doug Wendel
Professional Research Associate
University of Colorado at Boulder
Department of Chemistry and Biochemistry, Knight Lab
303-641-5442
wendel@colorado.edu

File: www/americangut/FAQ.psp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                        <h3 class="faq_header" id="faq1">Is my password the same as my verification code?</h3>
+                        <div class="faq_content">
+                            <p>No. Your <strong>password</strong> is printed on the sheet that you received with your kit in the mail. That sheet looks like this:</p><img src="img/creds_example.png"/><p>Your <strong>verification code</strong> is emailed to you.</p>
+                        </div>
+                        <div class="separator"></div>
+                        <h3 class="faq_header" id="faq2">My password doesn't work!</h3>
+                        <div class="faq_content">
+                            <p>The passwords have some ambiguous characters in them, so we have this guide to help you decipher which characters are in your password.</p>
+                            <p class="ambig">
+                            abcdefghijklmnopqrstuvwxyz<br>
```
Comment (ElDeveloper) [inline www/americangut/FAQ.psp:50]: I thought the `www/americangut/img/ambiguous_chars.png` resource would be used in this section, which makes more sense (I think).

**Comment by ElDeveloper** [comment]
Looking great, only thing that I noticed that might be annoying:
- Bullet-points are indented to the left whilst the text is centered.
- For the add participants workflow it would be cool if the overview steps (the bullet-points) were numbered so it gave a hint that the steps must be subsequent.

Unless anyone has any objection that would be everything before a good ol' :shipit:

**Comment by megalodonatron** [comment]
The bullet points are actually fine, you need to touch the css. Same goes for the ambiguous chars. Changed the add participant workflow to ordered list with numbers.

File: www/americangut/FAQ.psp (hunk 1 above)
Comment (megalodonatron) [inline www/americangut/FAQ.psp:50]: If you touch the css then the text here will be in the font that we used for printing etc

**Comment by ElDeveloper** [comment]
VirtualBooooooox _shakes fist_ :+1: 

This is looking beautiful, thanks @meganap!! And sorry for any inconvenience.

**Pull request merged**